index.html

```diff
@@ -178,12 +195,24 @@ <h1>Event Schedule</h1>
             <a href="javascript:void(0)" class="closebtn" onclick="closeNav('myNav4')">&times;</a>
             <div class="image_card4"></div>
             <div><h1>Frequently Asked Questions!</h1>
-                <h2>The Burning Question: What about COVID-19?</h2>
-                <p>UWB Hacks the Cloud will be happening on April 17, 2020 regardless!</p>
-                <p><strong>All workshops, technical talks, and presentations will be facilitated through Zoom.</strong> The Zoom invites will be available to all registered attendees in early April.</p>
-                <p><strong>All registered participants are welcome to attend the hackathon virtually.</strong> In the event that we can host the event on-campus, we will arrange for hybrid attendance so people can participate remotely.</p>
-                <p>The organizers are currently working with UW Bothell administrators to arrange for an in-person hackathon for those who wish to physically attend, assuming that the school okays it. <strong>We hope that this will be the case, and we will be updating all participants and volunteers about how campus closures or restrictions might affect the event.</strong></p>
-                <p>We will work hard to make sure you can participate regardless of whether you are self-quarantining or otherwise concerned about getting sick. Our collective health is our most valuable possession.</p>
+                <h2>What about COVID-19?</h2>
```

Comment:
might want to add a tidbit about how swag/goodies will work given that this is now all virtual